app/adachi/todo/page.tsx

```diff
@@ -0,0 +1,100 @@
+'use client'
+import { useState } from 'react'
+
+interface AppProps {
+  initialText?: string
+}
+
+const App = ({ initialText = '' }: AppProps): JSX.Element => {
+  const [text, setText] = useState<string>(initialText)
```

Comment:
コンポーネントの定義方法には、様々ありますが
このリポジトリでは、`export default function ComponentName() {`に統一しましょうか

また、コンポーネントの名前は、コンポーネントの役割や内容を踏まえた名前にしてほしいです。今回の場合は、`Todo`とかですね。

`initialText`は、以下のように書いた方はすっきりすると思います。
```suggestion
export default function Todo() {
  const [text, setText] = useState<string>('')
```